src/Microsoft.DotNet.XHarness.iOS.Shared/TestImporter/Templates/Managed/Resources/src/tvOSApp/ViewController.cs

```diff
@@ -105,13 +105,14 @@ public async override void ViewDidLoad()
             Xamarin.iOS.UnitTests.TestRunner.Jargon jargon = Xamarin.iOS.UnitTests.TestRunner.Jargon.NUnitV3;
```

Comment:
Is there a different default here?

Reply:
No, is NUnitV3, but we are passing via the args xUnit, should not matter for the manage packaged apps.